package.json

```diff
@@ -182,7 +182,7 @@
 		"mdast-util-to-string": "^2.0.0",
 		"moize": "^6.1.0",
 		"next": "^13.0.5",
-		"next-auth": "^4.17.0",
+		"next-auth": "^4.19.2",
```

Comment:
This fixed an issue with a transitive dependency that didn't allow node 18 in its engines.